shells/chrome/src/main.js

```diff
@@ -21,9 +21,6 @@ function createPanelIfReactLoaded() {
   chrome.devtools.inspectedWindow.eval(`!!(
     Object.keys(window.__REACT_DEVTOOLS_GLOBAL_HOOK__._renderers).length || window.React
   )`, function(pageHasReact, err) {
-    if (!pageHasReact || panelCreated) {
```

Comment:
Yes, it is guarding against [`onNavigated`](https://github.com/Tushkiz/react-devtools/blob/195ecd46e2fb1fbbf1bf7afd392f8f53c06fd2ea/shells/chrome/src/main.js#L46-L48) and [`setInterval`](https://github.com/Tushkiz/react-devtools/blob/195ecd46e2fb1fbbf1bf7afd392f8f53c06fd2ea/shells/chrome/src/main.js#L52-L54) callbacks